packages/swarm-constants/build/css/classnames.css

```diff
@@ -0,0 +1,208 @@
+.color-merlot {
+    color: #7A1D47;
+}
+
+.color-meetup-red {
+    color: #FF1154;
+}
+
+.color-peach {
+    color: #F65858;
+}
+
+.color-marigold {
+    color: #FFAD43;
+}
+
+.color-indigo {
+    color: #00455D;
+}
+
+.color-viridian {
+    color: #0098AB;
+}
+
+.color-beach {
+    color: #97CAD1;
+}
+
+.color-old-gold {
+    color: #947F5F;
+}
+
+.color-white {
+    color: #FFFFFF;
+}
+
+.color-alert-red {
+    color: #C80000;
+}
+
+.color-beach-highlight {
+    color: #F7FEFF;
+}
+
+.color-tooltip-blue {
+    color: #3679E4;
+}
+
+.color-gray-1 {
+    color: #F6F7F8;
+}
+
+.color-gray-2 {
+    color: #E6E6E6;
+}
+
+.color-gray-3 {
+    color: #D9D9D9;
+}
+
+.color-gray-4 {
+    color: #C5C4C4;
+}
+
+.color-gray-5 {
+    color: #A2A2A2;
+}
+
+.color-gray-6 {
+    color: #585858;
+}
+
+.color-gray-7 {
+    color: #212121;
+}
+
+.color-font-base {
+    color: #F6F7F8;
+}
+
+.color-font-secondary {
+    color: #C5C4C4;
+}
+
+.color-font-tertiary {
+    color: #212121;
+}
+
+.font-family-normal {
```

Comment:
This is running all all the properties.json files. I need to figure out how to update `config.json` so that it only does this for the color properties.

Reply:
This was fixed by using `registerFilter`: https://github.com/meetup/swarm-ui/pull/45/files#diff-554732290b7e20eb9f730fd659234489R15